index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Portfolio website">
+  <meta name="keywords" content="HTML, CSS">
+  <meta name="author" content="Cedric KOSSI">
   <link rel="stylesheet" href="main.css">
   <title>Cedric KOSSI</title>
 </head>
+
 <body>
-<h1>Hello Microverse!</h1>
+  <div class="navbar">
+    <div class="container">
+      <a class="logo" href="#">My Logo</a>
+      <img src="img/Union.png" alt="Navigation">
+    </div>
+  </div>
+  <section class="hero">
+    <div class="container">
```

Comment:
Kindly add the background image for the hero section, as the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/153531140-7996f671-8fbf-4e05-bc09-00f9df391662.png) | ![image](https://user-images.githubusercontent.com/35535130/153531191-29241bbd-524a-4d13-905f-413e1e1762e2.png) |